extensions/interactions/AlgebraicExpressionInput/directives/algebraic-expression-input-interaction.component.html

```diff
@@ -1,5 +1,5 @@
-<div class="guppy-div-learner protractor-test-guppy-div" ng-click="$ctrl.showOSK()"></div>
+<div class="guppy-div-learner protractor-test-guppy-div" (click)="showOsk()"></div>
 
-<div ng-show="!$ctrl.isCurrentAnswerValid()" class="alert alert-danger mt-2 oppia-algebraic-warning">
-  <[$ctrl.warningText]>
+<div *ngIf="!isCurrentAnswerValid()" class="alert alert-danger mt-2 oppia-algebraic-warning">
```

Comment:
Shouldn't we be using `[hidden]` instead?

Reply:
I didn't see any UI difference with ngIf. The regression comes when the UI shifts down because this node gets created. However, I didn't see any of this when checking the changes.

Reply:
Ah, I see, thanks for clarifying!